apps/src/utils.js

```diff
@@ -110,14 +111,16 @@ export function executeIfConditional(conditional, fn) {
  * @param inputString
  * @returns {string} string without quotes
  */
-export function stripQuotes(inputString) {
+export function stripQuotes(inputString:string):string {
   return inputString.replace(/["']/g, "");
 }
 
+
 /**
  * Defines an inheritance relationship between parent class and this class.
  */
-Function.prototype.inherits = function (parent) {
+// $FlowFixMe: extending built-in object prototypes is a no-no.
+Function.prototype.inherits = function (parent:Function) {
```

Comment:
yeah, this is imposed by flow. I'm pretty sure there is an eslint rule you could turn on to do the same thing though, since IMO this is an antipattern whether or not you are using flow.